testing/test_pytest_html.py

```diff
@@ -30,20 +27,20 @@ def read_html(path):
 
 def assert_results_by_outcome(html, test_outcome, test_outcome_number, label=None):
     # Asserts if the test number of this outcome in the summary is correct
-    regex_summary = r"(\d)+ {0}".format(label or test_outcome)
+    regex_summary = r"(\d)+ {}".format(label or test_outcome)
     assert int(re.search(regex_summary, html).group(1)) == test_outcome_number
 
     # Asserts if the generated checkbox of this outcome is correct
     regex_checkbox = (
         '<input checked="true" class="filter" '
-        'data-test-result="{0}"'.format(test_outcome)
+        'data-test-result="{}"'.format(test_outcome)
     )
     if test_outcome_number == 0:
         regex_checkbox += ' disabled="true"'
     assert re.search(regex_checkbox, html) is not None
 
     # Asserts if the table rows of this outcome are correct
-    regex_table = 'tbody class="{0} '.format(test_outcome)
+    regex_table = f'tbody class="{test_outcome} '
```

Comment:
Not your fault, but isn't this missing a `"` ?
```suggestion
    regex_table = f'tbody class="{test_outcome}"'
```

Reply:
Could be. Changing it fails tests, so should be dealt with separately.